dnf-behave-tests/common/cmd.py

```diff
@@ -18,11 +18,23 @@ def when_I_execute_command_in_directory(context, command, directory):
     run_in_context(context, command.format(context=context), cwd=directory.format(context=context))
 
 
+@behave.given("I successfully execute \"{command}\" in \"{directory}\"")
```

Comment:
This should be for all steps, no just `.given`.

In the past I advocated against the "I successfully execute" step, because it's not aligned with what I expect from behave. It merges given/when steps together with then. I would prefer to keep it as it is.

Reply:
> This should be for all steps, no just `.given`.

I can do that :slightly_smiling_face: 

> In the past I advocated against the "I successfully execute" step, because it's not aligned with what I expect from behave. It merges given/when steps together with then. I would prefer to keep it as it is.

Though I don't really understand this paragraph... Should I also change the other "successfully" variations of the steps (those that are already there) to work for all behave steps? To be consistent and so that people don't get confused that some work for `Then` and some don't?